recipe/build.py

```diff
@@ -300,15 +299,19 @@ def __init__(self, plat, version, version_patch):
         self.nvtoolsext_fmt = "lib{0}.so*"
         self.nvtoolsextpath = None
         self.libdir = "lib"
+        self.machine = platform.machine()
 
-        if platform.machine() == "ppc64le":
+        if self.machine == "ppc64le":
             # Power 8 Arch
+            cuda_libs = ["accinj64", "cuinj64"]
```

Comment:
Why did we separate these down into architecture specific blocks instead of keeping it where it previously was?

Reply:
This was an ARM fix. The SBSA run file didn't have these libs